addons/base/views.py

```diff
@@ -331,7 +331,7 @@ def get_auth(auth, **kwargs):
     # TODO: Add a signal here?
     if waffle.switch_is_active(features.ELASTICSEARCH_METRICS):
         user = auth.user
-        if isinstance(node, Preprint) and not node.is_contributor(user):
+        if isinstance(node, Preprint) and not node.is_contributor_or_group_member(user):
```

Comment:
Even though preprints don't have OSF Groups at the moment, they may one day, and this allows that possibility.  For now, it will work for contributors only.